src/migrations/20220425000000-goal-templates.js

```diff
@@ -0,0 +1,1964 @@
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Goals', 'lastCeasedSuspendedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Goals', 'firstClosedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Goals', 'lastClosedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Goals', 'firstCompletedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Goals', 'lastCompletedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'firstNotStartedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'lastNotStartedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'firstInProgressAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'lastInProgressAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'firstCompleteAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'lastCompleteAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'firstSuspendedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('Objectives', 'lastSuspendedAt', {
+          type: Sequelize.DATE,
+          allowNull: true,
+        }, { transaction });
+      } catch (err) {
+        console.error(err); // eslint-disable-line no-console
+        throw (err);
+      }
+
+      // Enable logging while doing structural updates
+      try {
+        await queryInterface.sequelize.query(
+          `
+          SELECT "ZAFSetTriggerState"(null, null, null, 'ENABLE');
+          `,
+          { transaction },
+        );
+      } catch (err) {
+        console.error(err); // eslint-disable-line no-console
+        throw (err);
+      }
+    },
+  ),
+  down: async (queryInterface) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      await queryInterface.dropTable('GoalTemplates', { transaction });
```

Comment:
Should we throw an error if `down` is ran? With how big this migration is I doubt we will be able to directly roll back, so might as well error out and tell the person running the migration to reload the DB from a backup if they want to roll back.